zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/SnapshotVersion.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.snapshots;
+
+public enum SnapshotVersion {
```

Comment:
:thought_balloon: I'm a bit conflicted about naming here. I'd expect a "version" to be just something like 1, 2, ....
On the other hand I like the descriptive naming (i.e. "CHECKSUM_FROM_DB"). Maybe the field should not be called `version`/`checksumVersion` then?

Reply:
You're right, now that I understand more this naming scheme is pretty terrible even something like `ChecksumGenerationMethod` would be better. Will change this